main.py

```diff
@@ -417,6 +417,15 @@ def get(self, class_name, period):
 
         # schedule = self.get_schedule(self.request.get('id'))
         self.response.write(json.dumps(self.get_class_schedule(class_name, period)))
+    def strip_punctuation(self, text):
+        punctuation = set(string.punctuation + " ")
+        clean_text = "";
+        for character in text:
+            if character not in punctuation:
```

Comment:
There is no string.alphanum, you would have to do string.letters + string.numbers. Because there's no clean way to do it, I'm keeping it the way it is